CHANGELOG.md

```diff
@@ -72,6 +72,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
+- Added: Blast Shield Lock-On preset option
+- Added: Patch to fix Kerry Ann Odem's name in the credits on game version(s) where it is incorrectly spelled
+- Fixed: Main Plaza vault ledge door is now affected by door lock rando
+- Fixed: Elite Research Phazon Elite getting stuck on frame perfect cutscene skip
+- Fixed: Minor music transition fixes
+- Fixed: Chaos option export error when submerging rooms
+- Changed: [PAL] Remove extra door lock in Central Dynamo
+- Changed: [PAL] Load Hunter Metroid in Frost Cave
```

Comment:
Now you merged main but as I expected it is under the headline of the already released version 9.1.0 :p